wrappers/src/marshalling.hpp

```diff
@@ -523,6 +523,11 @@ class Utf16StringAccessor {
     {
         return std::string(m_data.get(), m_size);
     }
+    
+    std::string_view to_string_view() const
+    {
+        return std::string_view(m_data.get(), m_size);
+    }
```

Comment:
We could convert that to an operator so that `Utf16StringAccessor` gets implicitly converted.

Reply:
Is this suggestion going to be picked up?

Reply:
I've done it in https://github.com/realm/realm-dotnet/pull/3118, so let's merge this as is.